config/locales/es/settings.yml

```diff
@@ -125,6 +125,9 @@ es:
       valuation_comment_notification: "Notificar comentarios de evaluación"
       valuation_comment_notification_description: "Envía un email a todos los usuarios menos al que haya comentado asociados a un presupuesto participativo cuando se cree un nuevo comentario de evaluación"
       graphql_api: "API GraphQL"
+      dashboard:
+        notification_emails: "Emails del Panel de Progreso"
+        notification_emails_description: "Activar el envio de emails de notificaciones a los autores de las propuestas en la sección de Panel de Progreso"
```

Comment:
It looks like an accent is missing in "envío".

Althoug we don't seem to have a standard name in Spanish, it looks like in other places we write "panel de progreso", using lowercase :thinking:.